client/src/modules/Teams/components/MyTeamSection/MyTeamSection.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button, notification, Row, Space, Typography } from 'antd';
+import { EditTwoTone, CopyOutlined } from '@ant-design/icons';
+import { useMemo } from 'react';
+import { TeamApi, TeamDistributionDetailedDto, TeamDistributionStudentDto, TeamDto } from 'api';
+import StudentsTable from '../StudentsTable/StudentsTable';
+import { useCopyToClipboard } from 'react-use';
+
+const { Text, Title } = Typography;
+
+type Props = {
+  distribution: TeamDistributionDetailedDto;
+  setTeamData: React.Dispatch<React.SetStateAction<Partial<TeamDto> | null>>;
+  studentId?: number;
+  copyPassword: (teamId: number) => Promise<void>;
+  reloadDistribution: () => Promise<void>;
+  setActiveTab: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const teamApi = new TeamApi();
+
+export default function MyTeamSection({
+  distribution,
+  setTeamData,
+  studentId,
+  copyPassword,
+  reloadDistribution,
+  setActiveTab,
+}: Props) {
+  const myTeam = distribution.myTeam!;
```

Comment:
consider fixing types, so the component should receive proper `myTeam` to avoid non null assertion

Reply:
fixed